.github/pull_request_template.md

```diff
@@ -1,18 +1,86 @@
-Describe your changes here.
+## Summary
```

Comment:
Might be redundant as a title, the first couple of lines should summarize the PR (if applicable)